src/model/docker.ts

```diff
@@ -89,6 +112,13 @@ class Docker {
             --volume "${actionFolder}/default-build-script":"c:/UnityBuilderAction" \
             --volume "${actionFolder}/platforms/windows":"c:/steps" \
             --volume "${actionFolder}/BlankProject":"c:/BlankProject" \
+            ${sshAgent ? `--volume ${sshAgent}:c:/ssh-agent` : ''} \
+                ${
+                  sshAgent && !sshPublicKeysDirectoryPath
+                    ? `--volume c:/Users/Administrator/.ssh/known_hosts:c:/root/.ssh/known_hosts`
```

Comment:
Could you explain where `c:/root` comes from? Have you tested that this works?

Reply:
I tested only for linux runner, tried to port to windows as best as I can...
but looking at it again seems that `sshAgent` isn't used in Windows anyway so probably I should remove these windows changes enitrely

Reply:
Yea better to have working software with missing features than the added complexity of a non working piece of which nobody knows how much of it was intended to work.

Thanks.